R/models.R

```diff
@@ -93,6 +93,20 @@ predict_model.lda <- function(x, newdata, type, ...) {
     prob = as.data.frame(res$posterior, check.names = FALSE)
   )
 }
+predict_model.H2OModel <- function(x, newdata, type, ...){
+    if (!requireNamespace('h2o', quietly = TRUE)) {
+        stop('The h2o package is required for predicting h2o models')
+    }
+    pred <- h2o.predict(x, h2o::as.h2o(newdata))
+    h2o_model_class <- class(x)[[1]]
+    if (h2o_model_class %in% c("H2OBinomialModel", "H2OMultinomialModel")) {
+        return(as.data.frame(pred[,-1])) 
+    } else if (h2o_model_class == "H2ORegressionModel") {
+        return(as.data.frame(pred[,1]))
```

Comment:
The column name for regression model output should be `'Response'`

Reply:
I'm making a change to the data frame column name to go from "predict" to "Regression". The regression model output will look like this after the change:
```
> predict_model(model.rf, newdata = df_h2o)
     Response
1   347.01963
2   332.42363
3   414.32455
4   373.05091
5   350.00000
6   385.49418
7   406.14417
8   415.23508
9   419.20509
10  381.11054
```